cups/adminutil.c

```diff
@@ -775,7 +782,7 @@ cupsAdminSetServerSettings(
       if (debug_logging)
       {
         cupsFilePuts(temp,
-	             "# Show troubleshooting information in error_log.\n");
+	             "\n# Show troubleshooting information in error_log.\n");
```

Comment:
Since we preserve the comments, we should adjust the line, remove it from here and move it to default cupsd.conf - otherwise we get duplicity.

A new comment in default cupsd.conf can look like:

```
# Set LogLevel to debug for turning on troubleshooting
```